index.js

```diff
@@ -175,6 +167,7 @@ instance.prototype.config_fields = function () {
 // When module gets deleted
 instance.prototype.destroy = function() {
 	var self = this;
+	clearInterval (heartbeat_interval); //Stop Heartbeat
```

Comment:
You should add this before setInterval too. Since each time a packet is received with self.login true, and self.socket.connected. You will start a new timer. So you are having an increasing amount of timers started. Just make sure you clear it, before you start a new one.

Reply:
@haakonnessjoen Thank-you,

I've still not got something quite right as the heatbeat is still active